GANDLF/utils/gandlf_logging.py

```diff
@@ -0,0 +1,61 @@
+import logging
+import yaml
+from pathlib import Path
+from importlib import resources
+import tempfile
+from GANDLF.utils import get_unique_timestamp
+
+
+def _create_tmp_log_file():
+    tmp_dir = Path(tempfile.gettempdir())
+    log_dir = Path.joinpath(tmp_dir, ".gandlf")
+    log_dir.mkdir(parents=True, exist_ok=True)
+    log_file = Path.joinpath(log_dir, get_unique_timestamp() + ".log")
+    return log_file
+
+
+def _create_log_file(log_file):
+    log_file = Path(log_file)
+    log_file.write_text("Starting GaNDLF logging session \n")
+
+
+def _save_logs_in_file(log_file, config_path):
```

Comment:
just a nitpick - this function is not saving any logs, instead just configures logging. Maybe rename it to smth more relevant? For ex,
```suggestion
def _configure_logging_with_logfile(log_file, config_path):
```